doc/man7/EVP_KDF-SSHKDF.pod

```diff
@@ -80,6 +80,22 @@ A single char of value 70 (ASCII char 'F').
 
 =back
 
+=item "fips-indicator" (B<OSSL_KDF_PARAM_FIPS_APPROVED_INDICATOR>) <int>
+
+A getter that returns 1 if the operation is FIPS approved, or 0 otherwise.
+This may be used after calling EVP_KDF_derive. It returns 0 if the related
+check parameter is set to 0.
+This option is used by the OpenSSL FIPS provider.
+
+=item "key-check" (B<OSSL_KDF_PARAM_FIPS_KEY_CHECK>) <int>
+
+The default value of 1 causes an error during EVP_KDF_derive() if
+the length of used key-derivation key is too short.
```

Comment:
should this be more specific... and refer to the B<OSSL_KDF_PARAM_KEY> needs to be at least 112 bits.

(Same comment applies to other pod files).

Reply:
The pod files updated.